card/spec/lib/card/set_pattern_spec.rb

```diff
@@ -44,13 +44,13 @@ def it_generates( opts )
 end
 
 describe Card::AllPlusSet do
-  it_generates :name => "*all plus", :from => Card.new( :name => "Book+author" )
+  it_generates name: "*all plus", from: Card.new( name: "Book+author" )
 end
 
 describe Card::AllSet do
-  it_generates :name => "*all", :from => Card.new( :type => "Book" )
+  it_generates name: "*all", from: Card.new( type: "Book" )
 end
 
 describe Card::TypePlusRightSet do
-  it_generates :name => "Book+author+*type plus right", :from => Card.new( :name=>"Iliad+author" )
+  it_generates name: "Book+author+*type plus right", from: Card.new( name: "Iliad+author" )
```

Comment:
Line is too long. [91/80]<br>Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.